openslides/assignments/static/js/assignments/site.js

```diff
@@ -34,10 +34,7 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
             .state('assignments.assignment.list', {
                 resolve: {
                     assignments: function(Assignment) {
-                        return Assignment.findAll();
-                    },
-                    items: function(Agenda) {
-                        return Agenda.findAll().catch(
+                        return Assignment.findAll().catch(
```

Comment:
this lines should be reverted (Assignments.findAll() does not need a catch block but the Agenda.findAll() need it)